server/app/models.py

```diff
@@ -1246,11 +1274,13 @@ def get_scores(self):
         else:
             members = [self.submitter]
         for member in members:
-            email = member.get().email[0]
+            mem_email = member.get().email[0]
+            grader_email = None
+            
             for score in self.submission.get().score:
                 all_scores.append([email,
                         score.score,
                         score.message,
-                        score.grader.get().email[0],
+                        grader_email,
                         score.tag])
```

Comment:
Yeah, not sure what the rationale behind it was. Not related to Luise's code, so I removed the change.